src/modules/core/components/BannedTag/BannedTag.css

```diff
@@ -0,0 +1,37 @@
+.icon {
+  display: inline-block;
+  margin-top: -4px;
+  margin-right: 4px;
+  margin-left: -4px;
```

Comment:
Please combine this into the shorthand version:

```suggestion
  margin: -4px 4px -4px 0;
```

Reply:
done!